extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/model/tasks/RememberTheMilkResponse.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.transfer.rememberthemilk.model.tasks;
+
+import com.google.api.client.util.Key;
+
+/** A generic response from the remember the milk service. */
+public abstract class RememberTheMilkResponse {
```

Comment:
can this class be package private? same goes for a bunch of them that I think are only used in this pkg

Reply:
Theyre all used in the RememberTheMilkService and throughout the importer and exporter, so they can't be package private unless we move those into the model package.